xbmc/settings/Settings.cpp

```diff
@@ -628,6 +630,23 @@ void CSettings::InitializeVisibility()
     timezone->SetRequirementsMet(false);
   }
 #endif
+  // hide Updates category in system section if there is no UpdaterImpl
+  if (!CUpdateHandler::GetInstance().UpdateSupported())
+  {
+    const auto systemSection = GetSettingsManager()->GetSection("system");
+    if (systemSection)
+    {
+      SettingCategoryList categories = systemSection->GetCategories();
+      for (auto category : categories)
+      {
+        if (category->GetId() == "updates")
+        {
+          category->SetRequirementsMet(false);
+          break;
+        }
+      }
+    }
+  }
```

Comment:
Yes but windows for example has only update capabilities when it's the desktop version - not the App Store version - hence the runtime check